src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1314,123 +1320,66 @@ void window_guest_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
     // Happiness
     gfx_draw_string_left(dpi, STR_GUEST_STAT_HAPPINESS_LABEL, nullptr, COLOUR_BLACK, screenCoords);
 
-    int32_t happiness = peep->Happiness;
-    if (happiness < 10)
-        happiness = 10;
-    int32_t ebp = COLOUR_BRIGHT_GREEN;
-    if (happiness < 50)
-    {
-        ebp |= BAR_BLINK;
-    }
-    window_guest_stats_bars_paint(happiness, screenCoords.x, screenCoords.y, w, dpi, ebp);
+    int32_t happiness = NormalizeGuestStatValue(peep->Happiness, PEEP_MAX_HAPPINESS, 10);
+    int32_t barColour = COLOUR_BRIGHT_GREEN;
+    bool barBlink = (happiness < 50) ? true : false;
```

Comment:
```suggestion
    bool barBlink = happiness < 50;
```
The `happiness < 50` will already return a bool, no need for the ternary.